python/cudf/cudf/_lib/json.pyx

```diff
@@ -156,7 +156,7 @@ def write_json(
     bool include_nulls=True,
     bool lines=False,
     bool index=False,
-    int rows_per_chunk=8,
+    int rows_per_chunk=1024*256,  # 256K rows
```

Comment:
Why it was set to `8` before? `8` rows per chunk is too small.

Reply:
CSV writer uses `8`, so used the same here!
https://github.com/rapidsai/cudf/blob/9c9862f97b00fc90760c3371f30ef5769f4163f5/python/cudf/cudf/_lib/csv.pyx#L476